src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -464,6 +466,25 @@ public Entity get(final ProjectileLaunchEvent e) {
 				return e.getEntity();
 			}
 		}, 0, "Use 'projectile' and/or 'shooter' in shoot events", ProjectileLaunchEvent.class);
+		//ProjectileCollideEvent
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.ProjectileCollideEvent")) {
+			if (Skript.methodExists(ProjectileCollideEvent.class, "getEntity"))
```

Comment:
Are these method checks necessary? If they are, you can just ignore this.

Reply:
These method checks are not necessary, they've been there since paper implemented this event. https://github.com/PaperMC/Paper/blob/ver/1.11.2/Spigot-API-Patches/0042-Add-ProjectileCollideEvent.patch

Reply:
Removed